src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -1,44 +1,50 @@
-<div class="sky-autocomplete">
+<div
+  aria-autocomplete="list"
+  aria-haspopup="true"
+  class="sky-autocomplete"
+  role="combobox"
+  [attr.aria-expanded]="isOpen"
+  [attr.aria-owns]="(isOpen) ? resultsListId : null"
+  [attr.id]="resultsWrapperId"
+>
   <ng-content>
   </ng-content>
-  <div class="sky-autocomplete-results">
-
-    <sky-dropdown
```

Comment:
Why remove the dropdown?

FYI- the `country-field-component` has some specific styles that are using dropdown selectors, and it looks like this change makes the flag on its own line:
<img width="295" alt="Screen Shot 2020-04-06 at 11 43 38 AM" src="https://user-images.githubusercontent.com/18354182/78578561-b1b3c000-77fd-11ea-8014-955b0ef00385.png">

Reply:
RE: "Why remove the dropdown?"
The intent of this pull request is to remove a needless dependency on the popovers module. The autocomplete has specific use-cases that don't apply to dropdown menus, etc.

Reply:
Good catch with the country field styles.